### PR TITLE
Update gm_global_a_lowspeed.dbc to include BOLT EUV Blindspot indicator

### PR DESCRIPTION
**Pull request by ServerDummy**
https://github.com/commaai/opendbc/assets/63013446/34f7537b-125b-4b43-85e7-7cacbce6df41

I added the bits for the blind spot monitoring light indicator, The message has some more info in it as well that I could not figure out exactly what it's for. Captured messages using a White panda hooked up to the aux port of my c3

### CODE REVIEW

**Comment by morrislee** [comment]
Looking forward if we can pick this up with the built in panda in C3 or C3x

File: gm_global_a_lowspeed.dbc [hunk 1]
```diff
@@ -89,6 +89,12 @@ BO_ 276135936 CruiseButtons: 3 GMLAN
 BO_ 276127744 CruiseButtons2: 1 GMLAN
  SG_ LKAGapButton : 1|2@0+ (1,0) [0|2] ""  NEO
 
+BO_ 275955897 LeftRadar: 2 GMLAN
+ SG_ BSM_Indicator_Light : 4|1@0+ (1,0) [0|0] "" XXX
+
+BO_ 275980379 RightRadar: 2 GMLAN
+ SG_ BSM_Indicator_Light : 0|1@0+ (1,0) [0|0] "" XXX
```
Comment (sshane) [inline gm_global_a_lowspeed.dbc:96]:
```suggestion
 SG_ BSM_Indicator_Light : 4|1@0+ (1,0) [0|1] "" XXX

BO_ 275980379 RightRadar: 2 GMLAN
 SG_ BSM_Indicator_Light : 0|1@0+ (1,0) [0|1] "" XXX
```

**Comment by sshane** [comment]
Have an example of the right radar working?

**Comment by ServerDummy** [comment]
Not as well synced as the first vid my apologies. 
https://github.com/commaai/opendbc/assets/63013446/95f5ba1d-59c0-4972-8498-c73f5c6f8e9d

**Pull request merged**